tests/integration/security/ca_custom_root/trust_domain_alias_client_test.go

```diff
@@ -0,0 +1,182 @@
+package cacustomroot
+
+import (
+	"fmt"
+	"istio.io/istio/pkg/config/protocol"
+	"istio.io/istio/pkg/test"
+	"istio.io/istio/pkg/test/echo/common"
+	"istio.io/istio/pkg/test/echo/common/scheme"
+	"istio.io/istio/pkg/test/env"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/tests/integration/security/util/cert"
+	"istio.io/istio/tests/integration/security/util/connection"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+const (
+	HTTPS  = "https"
+	POLICY = `
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
+metadata:
+  name: "mtls"
+spec:
+  mtls:
+    mode: STRICT
+---
+apiVersion: "networking.istio.io/v1alpha3"
```

Comment:
I believe this is not needed. Have you tried running test without ISTIO_MUTUAL?

Reply:
I didnt, will try and add here.

Reply:
getting error without the destination rule.